packages/files-ui/cypress/support/commands.ts

```diff
@@ -143,6 +150,31 @@ Cypress.Commands.add("safeClick", { prevSubject: "element" }, ($element?: JQuery
     .should($el => expect($el).to.not.be.visible)
 })
 
+Cypress.Commands.add("iframeLoaded", { prevSubject: "element" }, ($iframe?: JQuery<HTMLElement>): any => {
+  const contentWindow = $iframe?.prop("contentWindow")
+  return new Promise(resolve => {
+    if (
+      contentWindow &&
+              contentWindow.document.readyState === "complete"
+    ) {
+      resolve(contentWindow)
+    } else {
+      $iframe?.on("load", () => {
+        resolve(contentWindow)
+      })
+    }
+  })
+})
+
+Cypress.Commands.add("getInDocument", { prevSubject: "document" }, (document: any, selector: keyof HTMLElementTagNameMap) =>
+  Cypress.$(selector, document))
+
+Cypress.Commands.add("getWithinIframe", (targetElement: any, selector: string) =>
+  cy.get(selector || "iframe")
```

Comment:
we should add the wait here, rather than hardcoding it anywhere else.
I believe doing something like this should work
```suggestion
  cy.get(selector || "iframe", { timeout: 10000 })
```

Reply:
Thank you. I added it but when running repeatedly still got some issues when I removed the 1 second waits. 

I _think_ because there is a slight delay from when it becomes visible to when it becomes interactive (these iframe elements specifically).